LayoutTests/media/track/texttrackcue/texttrackcue-constructor-expected.txt

```diff
@@ -2,7 +2,7 @@
 PASS TextTrackCue should throw "TypeError" if the cue node is not a documentFragment
 PASS TextTrackCue should throw "InvalidNodeTypeError" if passed an invalid documentFragment
 PASS TextTrackCue should throw "InvalidNodeTypeError" if documentFragment has an invalid node
-PASS TextTrackCue should throw "InvalidStateError" if documentFragment doesn't have 'cuebackground' and 'cue' nodes
-PASS TextTrackCue should allow all valid node types
+PASS TextTrackCue should throw "InvalidNodeTypeError" if documentFragment doesn't have 'cuebackground' and 'cue' nodes
+FAIL TextTrackCue should allow all valid node types Invalid node type: IMG
```

Comment:
We should remove `img` from the list of valid node types in this test

Reply:
oh oops! well spotted!